lib/jekyll/assets/proxies/magick.rb

```diff
@@ -9,10 +9,9 @@
   presets = %W(@2x @4x @1/2 @1/3 @2/3 @1/4 @2/4 @3/4
     @double @quadruple @half @one-third @two-thirds @one-fourth
       @two-fourths @three-fourths)
+  ARGS=args.freeze
```

Comment:
@envygeeks it was originally in code https://github.com/jekyll/jekyll-assets/blob/master/lib/jekyll/assets/proxies/magick.rb#L15 - frankly to say I have no idea what's the idea behind it and why it was done like that to be used like this https://github.com/jekyll/jekyll-assets/blob/master/lib/jekyll/assets/proxies/magick.rb#L76